contents/teams/web-analytics/objectives.mdx

```diff
@@ -1,21 +1,20 @@
-## Q3 2025 Goals
+## Q4 2025 Goals
 
-* Continue to improve pre-aggregation <TeamMember name="Lucas Ricoy" photo />
-  * Ensure the data is accurate
-  * Make the dagster jobs reliable
-  * Automatically roll this out to customers above a given size (e.g. 500k pageviews a day)
-* Contribute to the B2B2C project team <TeamMember name="Lucas Ricoy" photo /> & <TeamMember name="Robbie Coomber" photo />
-  * Build external web analytics api that can be queried across a very large number of projects
-  * Work closely with a specific customer to ensure we meet their needs
-  * Speak to other tools to see if they would be open to integrating with us <TeamMember name="Anna Szell" photo />
 * Marketing analytics <TeamMember name="Javier Bahamondes" photo />
-  * Finish everything needed for beta readiness, start public beta
-  * Build DWH integrations for our marketing team to dogfood
-* Cookieless mode <TeamMember name="Robbie Coomber" photo />
-  * Support error tracking and heatmaps
-  * Stretch goal: support session replay
-  * Invite beta users and make it generally available
-* Miscellaneous
-  * Update code structure to new product structure
-  * Max AI - Make Max aware of web analytics queries and dashboards
+  * Release marketing analytics as general available (GA)
+  * Add ROAS widget and at least two major integrations (Meta/YouTube)
+* Bring AI into web analytics <TeamMember name="Lucas Ricoy" photo /> & <TeamMember name="Javier Bahamondes" photo />
+  * Integrate web analytics with Max
```

Comment:
This is super vague - what are we actually going to do? Make sure Max has context about web analytics canned insights so he (it?) can use them in answering questions?

Reply:
I think:

- Make Max aware of WebAnalytics/session metrics
- Canned insights for both event and session-level metrics 
- Come up with filters like in Session Replay
- Stretch: Explain anomalies with deep research

Reply:
Cool, can you add them to the actual doc?

Reply:
@raquelmsmith yeah, done. The only missing one is the experiment definition now.